librz/util/buf.c

```diff
@@ -321,7 +365,15 @@ RZ_API RzBuffer *rz_buf_new_slice(RzBuffer *b, ut64 offset, ut64 size) {
 }
 
 // TODO: rename to new_from_file ?
-RZ_API RzBuffer *rz_buf_new_slurp(const char *file) {
+/**
+ * \brief Creates a new buffer from a file.
+ * \param file The filename used to create the new buffer.
+ * \return Return the new allocated buffer.
+ *
+ * The function creates a new allocated buffer using the RZ_BUFFER_BYTES back end.
+ * The buffer content isn't synchronized with file content.
```

Comment:
The function creates a new buffer in memory, initializing it with the whole content of the specified file.